api/openapi/my_api.yaml

```diff
@@ -225,263 +246,194 @@ components:
         - pluginType
-        - requiredPatientVariables
       properties:
-        title:
-          type: string
-          description: "Meaningful name for this plugin"
-          example: "Aminoglycoside dosing guidance"
         piid:
           type: string
           description: "A string, unique to this deployment, used to identify this plugin in realtime"
-          example: "pdspi-aminoglycoside-nomogram"
+          example: "pdspi-mapper-DOAC"
+        title:
+          type: string
+          description: "Meaningful name for this plugin"
+          example: "DOAC variable mapper"
+        pluginDependencies:
+          type: array
+          items:
+            type: string
+            description: "An array of container names upon which the plugin in question depends. For example, 'txscience/pds-fhir-example:0.2.0'. Mapper plugins must indicate a dependency on one or more FHIR plugins, but dependencies are discouraged otherwise in order to maintain the decoupling and reuse of plugins. Every plugin of type 'm' and 'mD' must have at least one 'f' or 'fD' plugin listed in pluginDependencies, otherwise return 500."
         pluginType:
           type: string
           enum: [e,g,m,mD,f,fD]
           description: "A value from the enumeration below identifying the plugin type. 'g'=Guidance, 'c'=convenience, 'm'=mapper, 'f'=FHIR data server, 'mD'= default mapper, 'fD'=default FHIR server. In the clinical arena, only one FHIR server and only one mapper is allowed, and this can be indicated either with the defaults, or by only installing one mapper and one FHIR plugin type."
-          example: "g"
+          example: "m"
         pluginTypeTitle:
           type: string
           description: "A human-readable value for the type. 'g'=Guidance, 'c'=Convenience, 'm'=Mapping, 'f'=FHIR, 'mD'=Default Mapping plugin, 'fD'=Default FHIR plugin"
-        pluginDependencies:
+        settingsDefaults:
+            $ref: '#/components/schemas/Settings'
+    Guidance:
+      description: "Returns guidance, justification, visualization data and data type, as well as cds-hooks compatible formats for direct integration with the EHR."
+      type: object
+      required:
+        - txid
+        - inputs
+        - cards
+      properties:
+        piid:
+          description: "A string, unique to this deployment, used to identify this plugin in realtime"
+          type: string
+          example: "pdspi-aminoglycoside-nomogram"
+        title:
+          description: "An optional title for the guidance plugin"
+          type: string
+          example: "Aminoglycoside dosing guidance"
+        txid:
+          type: string
+          description: "The transaction id sent in the GuidanceInputs. If none were sent, a unique id will be created and sent back here."
+          example: "38-1"
+        inputs:
+          type: object
+          items:
+            $ref: '#/components/schemas/Settings'
+        return:
+          type: object
```

Comment:
I don't think you need to have 'type' and 'items' here - it's just going to add more syntactical sugar to the example, I think.